### PR TITLE
PLAT-490: Pass instance to manifest generation

### PR DESCRIPTION
**Pull request by bjarneo**
* Add the instance as an input to use it for naming i.e. the ingress so it does not conflict with other ingresses in the same region
* Move the ingress generation to its own function
* Update the test

Example error from an ingress if another with identical name exist within the same namespace:
```bash
 Warning  FailedDeployModel  2s                     ingress  Failed deploy model due to DuplicateLoadBalancerName: A load balancer with the same name '<ingress-name>' exists, but with different settings
 ```

Ticket: https://dignio.atlassian.net/browse/PLAT-490?atlOrigin=eyJpIjoiYWIxYzY0N2Q5NWY5NDZlOGFjMzBkMzA3MTU3MTllNGUiLCJwIjoiaiJ9

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -46,11 +49,13 @@ runs:
   image: 'Dockerfile'
   args:
     - ${{ inputs.app_name }}
+    - ${{ inputs.service_type }}
```
Comment (bjarneo) [inline action.yaml:52]: This was missing 😬

**Comment by bjarneo** [comment]
Hm, have to solve the failing test. Works while testing locally.

**Review by weel — APPROVED** [review]
LGTM 💯 

Only nitpick or comment would be that the fixture had some indication that it's only used by test-action.yaml. 

Had to spend some extra brain power to figure this out.. 😅

**Comment by bjarneo** [comment]
> LGTM 💯
> 
> Only nitpick or comment would be that the fixture had some indication that it's only used by test-action.yaml.
> 
> Had to spend some extra brain power to figure this out.. 😅

Fair point. Will update the readme :)

**Pull request merged**